### PR TITLE
fix 2.0 api docs

### PR DESCRIPTION
**Pull request by zhupengyang**
### PR types

Others

### PR changes

Docs

### Describe

fix 2.0 api docs

- standard_normal
standard_normal

- normal
normal

- prelu fluid
prelu fluid

- LogSoftmax
image

- Hardtanh
image

- Hardsigmoid
image

- Hardshrink
image

- log_softmax
log_softmax

- leaky_relu
leaky_relu

- hardshrink
hardshrink

- gelu
gelu

- elu
elu

- zeros_like
zeros_like

- var
var

- uniform
uniform

- std
std

- randperm
randperm

- randn
randn

- randint
randint

- rand
rand

- ones_like
ones_like

- mean
mean

- arange
arange

### CODE REVIEW

**Review by hong19860320 — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -9734,12 +9734,10 @@ def swish(x, beta=1.0, name=None):
 @deprecated(since="2.0.0", update_to="paddle.nn.functional.prelu")
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py]: update_to='paddle.static.nn.prelu'

File: python/paddle/tensor/creation.py [hunk 2]
```diff
@@ -323,15 +320,13 @@ def ones_like(x, dtype=None, name=None):
...
 
     Examples:
         .. code-block:: python
 
             import paddle
 
-            paddle.disable_static()
-
             x = paddle.to_tensor([1,2,3])
             out1 = paddle.zeros_like(x) # [1., 1., 1.]
```
Comment (TCChenlong) [inline python/paddle/tensor/creation.py]: paddle.ones_like(x)

File: python/paddle/tensor/creation.py [hunk 3]
```diff
@@ -323,15 +320,13 @@ def ones_like(x, dtype=None, name=None):
...
             out2 = paddle.zeros_like(x, dtype='int32') # [1, 1, 1]
```
Comment (TCChenlong) [inline python/paddle/tensor/creation.py]: paddle.zeros_like(x, dtype='int32')

Comment (zhupengyang) [reply python/paddle/tensor/creation.py]: Done

File: python/paddle/tensor/creation.py (hunk 2 above)
Comment (zhupengyang) [reply python/paddle/tensor/creation.py]: Done

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (zhupengyang) [reply python/paddle/fluid/layers/nn.py]: Done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by zhiqiu — COMMENTED** [review]
LGTM, `paddle.fluid.layers.nn.prelu` is used in static graph.

**Review by zhiqiu — APPROVED** [review]

**Pull request merged**